### DISCUSSION TITLE
Canadian census fails to represent multiracial citizens

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#142] [General]
Twitter thread: https://twitter.com/Fumi_chun/status/1390016558211100677

tl;dr ALL non-white mixed people get put in the category "multiple visible minorities," people who are a minority mixed with white get put only in the minority tally

**Comment by albertkun** [comment]
@rh5140 I wonder what their reasoning for this was... Pretty disappointing though.. It really goes to show how things can go wrong when governments decide what is "normal" instead of people.
